skills/roborazzi/references/try_it_out.md

```
@@ -0,0 +1,26 @@
+<!-- Generated from docs/topics/try_it_out.md by the generateSkill task. Do not edit this file. -->
+# Try it out
+
+Available on Maven Central.
+
+### Add Robolectric
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)